pkg/alertmanager/clustertlsconfig/utils.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2024 The prometheus-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clustertlsconfig
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+	"k8s.io/utils/ptr"
+)
+
+func addServerTLSConfigToYaml(c ClusterTLSConfig, cfg yaml.MapSlice) yaml.MapSlice {
```

Comment:
why not this?
```suggestion
func (c *ClusterTLSConfig) addServerTLSConfigToYaml(cfg yaml.MapSlice) yaml.MapSlice {
```

Reply:
Hmm...It was initially a method of `ClusterTLSConfig`. It's been a while, not sure why I made the change now.